controllers/favoritesController.js

```diff
@@ -14,14 +15,33 @@ const create = (request, response) => {
                          response.status(200).send({message: `${request.body.location} has been added to your favorites`});
                      })
                      .catch((error) => {
-                         response.status(500).json({error});
+                         response.status(500).json({error})
                      })
             } else {
                 response.status(401).send({error: 'Unauthorized'})
             };
         })
 };
 
+const remove = (request, response) => {
+    database('users').where('apiKey', request.body.api_key)
+        .then((user) => {
+            if(user) {
+                database('favorites').where('userId', user[0].id).where('location', request.body.location).del()
+                    .then(() => {
+                        response.status(204).send({message: `${request.query.location} has been deleted`})
+                    })
+                    .catch((error) => {
+                        response.status(500).json({error})
+                    })
+            } else {
+                response.status(401).send({error: 'Unauthorized'})
+            };
+        })
+};
+
+
```

Comment:
Good logic here, I'd suggest moving some of it out to helper functions to help clean up the code.